charts/newrelic-infrastructure/templates/controlplane/agent-configmap.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.controlPlane.enabled -}}
+{{- if .Values.customAttributes | kindIs "string" }}
+{{- fail ( include "newrelic.compatibility.message.customAttributes" . ) -}}
```

Comment:
why having the fail here instead of having it unified in the notes.txt?

Reply:
I was desperately trying to make [this test work](https://github.com/newrelic/nri-kubernetes/blob/helm/adapt-helm-chart/charts/newrelic-infrastructure/tests/configmap_cp_agent_test.yaml#L110-L140) but I couldn't.

The thing here is that `helm unittest` thinks that the manifest you are trying to test is not in the testing suite if you fail in other manifests/files. So I wanted this very manifest to fail and properly test `failedTemplate` but I couldn't.

I would leave this as it is now until we find out how to test that the manifest fails correctly. WDYT?